src/elements/OcBreadcrumb.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="oc-breadcrumb">
+  <div class="oc-breadcrumb uk-overflow-hidden">
```

Comment:
Extend the `oc-breadcrumb` class pls with the overflow in its scss file - https://github.com/owncloud/owncloud-design-system/blob/ca7575202ef2009a6d603d660b43db89a3be4b61/src/styles/theme/oc-breadcrumb.scss#L6